internal/formats/common/cyclonedxhelpers/licenses.go

```diff
@@ -0,0 +1,24 @@
+package cyclonedxhelpers
+
+import (
+	"github.com/CycloneDX/cyclonedx-go"
+	"github.com/anchore/syft/internal/spdxlicense"
+	"github.com/anchore/syft/syft/pkg"
+)
+
+func Licenses(p pkg.Package) *cyclonedx.Licenses {
+	lc := cyclonedx.Licenses{}
+	for _, licenseName := range p.Licenses {
+		if value, exists := spdxlicense.ID(licenseName); exists {
```

Comment:
Interesting! I wasn't able to find a shared list between `spdx` and `cyclonedx` is this for convenience or did you see in their specification somewhere that they are doing licenses the same way?

Reply:
CycloneDX support SPDX license IDs and expressions along with unresolved license names. See: https://cyclonedx.org/use-cases/#license-compliance

Reply:
Thanks for the link @stevespringett!